cypress/e2e/cloud/operator.test.ts

```diff
@@ -0,0 +1,268 @@
+    // Validates that the default behavior is to open to the account tab
+    cy.getByTestID('accountTab').should('satisfy', element => {
+      const classList = Array.from(element[0].classList)
+      return classList.includes('cf-tabs--tab__active')
+    })
+
+    // Expect the org tab to be inactive
+    cy.getByTestID('orgTab').should('satisfy', element => {
+      const classList = Array.from(element[0].classList)
+      return classList.includes('cf-tabs--tab__active') === false
+    })
+
+    cy.get('.cf-refless-popover--trigger').click()
+
+    cy.getByTestID('refless-popover--contents').contains('test@influxdata.com')
+
+    cy.getByTestID('logout-button').should('exist')
+
+    // preloads 6 accounts
+    cy.getByTestID('table-body').within(() => {
+      cy.getByTestID('table-row').should('have.length', 6)
+    })
+
+    // filter the results by email
+    cy.getByTestID('operator-resource--searchbar')
+      .invoke('attr', 'placeholder')
+      .should('contain', 'Filter accounts by email')
+
+    cy.getByTestID('operator-resource--searchbar').type('asalem', {
+      force: true,
+      delay: 50,
+    })
+
+    cy.getByTestID('table-body').within(() => {
+      cy.getByTestID('table-row').should('have.length', 1)
+    })
+
+    // Make sure that the popover closes when clicking out
+    cy.getByTestID('refless-popover--contents').should('not.exist')
+
+    cy.getByTestID('operator-resource--searchbar')
+      .clear()
+      .type('salt_bae', {force: true, delay: 50})
+
+    cy.getByTestID('empty-state--text').contains(
+      'Looks like there were no accounts that matched your search'
+    )
+    cy.getByTestID('operator-resource--searchbar').clear()
+
+    cy.getByTestID('orgTab').click()
+
+    cy.location().should(loc => {
+      expect(loc.pathname).to.eq('/operator/orgs')
+    })
+
+    // Expect the org tab to be active
+    cy.getByTestID('orgTab').should('satisfy', element => {
+      const classList = Array.from(element[0].classList)
+      return classList.includes('cf-tabs--tab__active')
+    })
+
+    // Expect the account tab to be inactive
+    cy.getByTestID('accountTab').should('satisfy', element => {
+      const classList = Array.from(element[0].classList)
+      return classList.includes('cf-tabs--tab__active') === false
+    })
+
+    // preloads 6 organizations
+    cy.getByTestID('table-body').within(() => {
+      cy.getByTestID('table-row').should('have.length', 6)
+    })
+
+    // filter the results by email
+    cy.getByTestID('operator-resource--searchbar')
+      .invoke('attr', 'placeholder')
+      .should('contain', 'Filter organizations by id')
+
+    cy.getByTestID('operator-resource--searchbar').type('678', {
+      force: true,
+      delay: 50,
+    })
+    cy.getByTestID('table-body').within(() => {
+      cy.getByTestID('table-row').should('have.length', 1)
+    })
+
+    cy.getByTestID('operator-resource--searchbar')
+      .clear()
+      .type('invalid', {force: true, delay: 50})
```

Comment:
just curious what the `delay: 50` is for in all of these?

Reply:
omg so FOR SOME REASON when I would just set the `type('whatever')` most of the text would get trimmed. I tried a million different things (cy.focus, cy.should().type(), even a cy.wait thinking that the issue was that the DOM hadn't loaded) but in the end this is what I came from. I _think_ it has to do with the fact that we're debouncing the search text but i'm not :100: on that

Reply:
ohhhhh yeah the debounce thing actually makes sense. It probably confuses cypress into thinking it's done or something. Cool, nice fix!